rpcs3/rpcs3qt/settings_dialog.ui

```diff
@@ -54,7 +54,7 @@
        </sizepolicy>
       </property>
       <property name="currentIndex">
-       <number>0</number>
+       <number>1</number>
```

Comment:
Unnecessary change

Reply:
What does this one mean? Every single time I open the file with QT creator it changes it to a 1.

Reply:
It's the tab index you had open in the designer

Reply:
Ah, makes sense. GPU is tab 1 so every time I make a change I have to switch to CPU tab in the designer before I save. What a wonderful design.